server/src/admin/admin.router.ts

```diff
@@ -4,21 +4,27 @@ import { Auth } from '../auth/auth';
 import {
   AdminReviewRequestType,
   AdminRequestType,
+  AdminUserRequestType,
   RaffleWinnerRequestType,
   AdminAddSemesterRequestType,
   ReportReviewRequestType,
 } from './admin.type';
 import {
   getPendingReviews,
+  getReportedReviews,
+  getReviewCounts,
+  getCourseCSV,
   editReviewVisibility,
-  getRaffleWinner,
   removePendingReview,
   updateAllProfessorsDb,
   resetAllProfessorsDb,
   initAllDb,
   addNewSemDb,
   verifyTokenAdmin,
   reportReview,
+  getAdminUsers,
+  removeAdmin,
+  addOrUpdateAdmin,
 } from './admin.controller';
 
 export const adminRouter = express.Router();
```

Comment:
I think it would make it easier on the backend if all these routers were prepended by `/admin`. Like for getting reviews, that seems like it could belong to the display reviews page, but adding the admin route would help clarify that (and differentiate it from the user-facing reviews).